person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..2000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
In the method `of_age?`, kindly fix the typo by adding the @ before `age` on `line 15`.

The method `can_use_services?` is supposed to be a public method. Its position under the `private` keyword makes it private. You should place it above `line 12`. You can check this [link](https://www.rubyguides.com/2018/10/method-visibility/#:~:text=Every%20instance%20method%20after%20private%20becomes%20a%20private%20method.) for more info.